### PR TITLE
Bump bdk and bitcoin

### PR DESCRIPTION
**Pull request by bonomat**
bdk bump to 0.19
bitcoin bump to 0.28.1
secp256k1-zkp to 0.6

### CODE REVIEW

**Comment by bonomat** [comment]
supersedes https://github.com/comit-network/maia/pull/37 https://github.com/comit-network/maia/pull/32 https://github.com/comit-network/maia/pull/26

**Review by klochowicz — APPROVED** [review]
thanks for tackling this!

File: maia/Cargo.toml [hunk 1]
```diff
@@ -5,13 +5,13 @@ edition = "2021"
 
 [dependencies]
 anyhow = "1"
-bdk = { version = "0.18", default-features = false }
+bdk = { version = "0.19", default-features = false }
 itertools = "0.10"
-maia-core = { git = "https://github.com/comit-network/maia", rev = "e419cb2cf19bed6ec53eaa3672408a6205a9b705", package = "maia-core" } # Pin the maia-core version to before breaking change
```
[the remark this answers is not in the document]
Comment (bonomat) [reply maia/Cargo.toml:10]: I believe you are right, this upgrade is a breaking change and we will lose backwards compatibility unless we upgrade the old version as well.

[the remark this answers is not in the document]
Comment (luckysori) [reply maia/Cargo.toml:10]: Can someone clarify exactly what we are breaking with this change?

If we're breaking something does that mean we should bump `maia` to 0.3.0? This also makes me think we should add a changelog to this repository.

**Review by luckysori — COMMENTED** [review]
If we're breaking things I think we should remember to bump the version. A changelog would help us keeping track of this stuff.

**Comment by klochowicz** [comment]
> If we're breaking things I think we should remember to bump the version. A changelog would help us keeping track of this stuff.

the whole idea is that this thing was not supposed to be broken (that's why we didn't mere this PR yet) - I'll see whether we can get around it.

**Comment by klochowicz** [comment]
Superceded by #41

**Pull request closed without merging**